src/hotspot/share/jvmci/vmStructs_jvmci.cpp

```diff
@@ -776,53 +765,8 @@
-  declare_constant(VM_Version::CPU_SSE2)                            \
-  declare_constant(VM_Version::CPU_SSE3)                            \
-  declare_constant(VM_Version::CPU_SSSE3)                           \
-  declare_constant(VM_Version::CPU_SSE4A)                           \
-  declare_constant(VM_Version::CPU_SSE4_1)                          \
-  declare_constant(VM_Version::CPU_SSE4_2)                          \
-  declare_constant(VM_Version::CPU_POPCNT)                          \
-  declare_constant(VM_Version::CPU_LZCNT)                           \
-  declare_constant(VM_Version::CPU_TSC)                             \
-  declare_constant(VM_Version::CPU_TSCINV)                          \
-  declare_constant(VM_Version::CPU_AVX)                             \
-  declare_constant(VM_Version::CPU_AVX2)                            \
-  declare_constant(VM_Version::CPU_AES)                             \
-  declare_constant(VM_Version::CPU_ERMS)                            \
-  declare_constant(VM_Version::CPU_CLMUL)                           \
-  declare_constant(VM_Version::CPU_BMI1)                            \
-  declare_constant(VM_Version::CPU_BMI2)                            \
-  declare_constant(VM_Version::CPU_RTM)                             \
-  declare_constant(VM_Version::CPU_ADX)                             \
-  declare_constant(VM_Version::CPU_AVX512F)                         \
-  declare_constant(VM_Version::CPU_AVX512DQ)                        \
-  declare_constant(VM_Version::CPU_AVX512PF)                        \
-  declare_constant(VM_Version::CPU_AVX512ER)                        \
-  declare_constant(VM_Version::CPU_AVX512CD)                        \
-  declare_constant(VM_Version::CPU_AVX512BW)                        \
-  declare_constant(VM_Version::CPU_AVX512VL)                        \
-  declare_constant(VM_Version::CPU_SHA)                             \
-  declare_constant(VM_Version::CPU_FMA)                             \
-  declare_constant(VM_Version::CPU_VZEROUPPER)                      \
-  declare_constant(VM_Version::CPU_AVX512_VPOPCNTDQ)                \
-  declare_constant(VM_Version::CPU_AVX512_VPCLMULQDQ)               \
-  declare_constant(VM_Version::CPU_AVX512_VAES)                     \
-  declare_constant(VM_Version::CPU_AVX512_VNNI)                     \
-  declare_constant(VM_Version::CPU_FLUSH)                           \
-  declare_constant(VM_Version::CPU_FLUSHOPT)                        \
-  declare_constant(VM_Version::CPU_CLWB)                            \
-  declare_constant(VM_Version::CPU_AVX512_VBMI2)                    \
-  declare_constant(VM_Version::CPU_AVX512_VBMI)                     \
-  declare_constant(VM_Version::CPU_HV)
+#define DECLARE_LONG_CPU_FEATURE_CONSTANT(id, name, bit) GENERATE_VM_LONG_CONSTANT_ENTRY(VM_Version::CPU_##id)
+#define VM_LONG_CPU_FEATURE_CONSTANTS CPU_FEATURE_FLAGS(DECLARE_LONG_CPU_FEATURE_CONSTANT)
```

Comment:
Missing `#undef DECLARE_LONG_CPU_FEATURE_CONSTANT`.

Reply:
So comment as for `DECLARE_INT_CPU_FEATURE_CONSTANT`.